src/test/ui/nll/closures-in-loops.stderr

```diff
@@ -13,17 +13,21 @@ error[E0499]: cannot borrow `x` as mutable more than once at a time
   --> $DIR/closures-in-loops.rs:13:16
    |
 LL |         v.push(|| x = String::new());
-   |                ^^ - borrows occur due to use of `x` in closure
-   |                |
-   |                `x` was mutably borrowed here in the previous iteration of the loop
+   |         -------^^-------------------
```

Comment:
The spans are a bit unfortunate.  Is there anything to do here?

Reply:
You mean the format of using "-------^^-------------------" is not clear?

Reply:
I just realised: the longer `---` is meant to point to `loop_terminator_location`.  It should point to some `}` below, shouldn't it?